e2e/resources/enums.ts

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+
+export enum workspaceAccessLevel {
```

Comment:
What's the rationale for moving to this file? I'm not sure I like putting all enums into a file named `enums`. It doesn't really carry much logical significance to group them all here - to me it feels a bit similar to putting all the classes in a file named `classes.ts`. I think I prefer how it was before.

Reply:
If you feel very strongly I won't block, but I don't think this organization makes sense. These being enums is an implementation detail.

I'd suggest at least splitting this into two files: `workspaces.ts`/`workspace-utils.ts` and `navigation.ts`. Alternatively, rename this file to something that pertains to the logical grouping of these enums: maybe something like `page-identifiers` or `labels`?